css/style.css

```diff
@@ -138,35 +138,73 @@ ul.CMenu li:hover ul.left {left: -150px}
-div#t div.TB_Separator {width: 1px; height: 22px; margin: 7px 0 0 5px; padding: 0; border: 0; background-color: #A0A0A0}
-
-div#t div#add {background: transparent url(../images/toolbar.png) no-repeat -24px center}
-div#t div#create {background: transparent url(../images/toolbar.png) no-repeat -48px center}
-div#t div#remove {background: transparent url(../images/toolbar.png) no-repeat -72px center}
-div#t div#start {background: transparent url(../images/toolbar.png) no-repeat -96px center}
-div#t div#pause {background: transparent url(../images/toolbar.png) no-repeat -120px center}
-div#t div#stop {background: transparent url(../images/toolbar.png) no-repeat -144px center}
-div#t div#moveu {background: transparent url(../images/toolbar.png) no-repeat -168px center}
-div#t div#moved {background: transparent url(../images/toolbar.png) no-repeat -192px center}
-div#t div#search {background: transparent url(../images/toolbar.png) no-repeat -216px center}
-div#t div#rss {background: transparent url(../images/toolbar.png) no-repeat -240px center}
-div#t div#setting {background: transparent url(../images/toolbar.png) no-repeat -264px center}
-
-div#t div#help {background: transparent url(../images/quest.gif) no-repeat 0px center}
-div#t div#plugins {background: transparent url(../images/plugin.png) no-repeat 0px center}
-
-#rc { float: right; }
-#go {background: transparent url(../images/go.gif) no-repeat 0px center; }
-#query { margin-top: 4px; }
-div#t a#mnu_go {margin: 3px 0 0 0; padding: 3px}
-div#t #ind {margin-top: 2px; background: transparent url(../images/ajax-loader.gif) no-repeat 0px center; width: 32px; margin-right: 2px; line-height: 26px; cursor: default; }
-
-input.Textbox {border: 1px solid #B0B0B0; font-size: 11px;  padding: 2px}
+div#t {
+  height: 36px;
+  background: #F0F0F0 url(../images/t_bg.png) repeat-x center 0.25rem;
+  border-bottom: 1px solid #D0D0D0;
+  white-space: nowrap;
+}
+#top-menu {
+  background-color: #F0F0F0;
+  border-bottom: 1px solid #D0D0D0;
+  z-index: 30;
+}
+div#t button.navbar-toggler {width: 3rem; background-color: #F0F0F0;}
+div#t a {padding: 3px;}
+div#t div.navbar-nav a:hover {
+  filter: brightness(1.3);
+}
+div#t div.navbar-nav a:active {padding: 4px 2px 2px 4px;}
+div#t div.TB_Separator {
+  align-self: stretch;
+  flex: 0 0 1px;
+  margin: 0.25rem;
+  padding: 0;
+  border: 0;
+  background-color: #A0A0A0;
+}
+div#t .nav-link {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 0.5rem;
+  font-weight: bold;
+  border: 1px solid #A0A0A0;
+  border-radius: 8px;
+  background-color: #F0F0F0;
+  background-image: none;
+  margin: 0.15rem;
+}
+div.navbar-nav div.nav-icon {
+  width: 24px;
+  height: 24px;
+  cursor: pointer;
+  background-color: transparent;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+div#add, div#remove, div#start, div#pause, div#stop, div#search, div#rss, div#setting {background-image: url(../images/toolbar.png);}
```

Comment:
Reduced the specificities on these selectors, in order that the plugin- and theme-specific rules can override them.